fix(chart): adjust scaling margins and visible range logic

valeryV99 opened the pull request:
- Added `scaleMargins` property to chart config for improved chart padding.
- Updated visible range logic to extend the `from` value for better time scale adjustment.
https://www.notion.so/254362757c98816eaa18db14ba3bcb0d?v=254362757c9881b38afd000c1ae9b764&p=25d362757c9880a5a717f50051740ee5&pm=s

valeryV99 committed:
fix(chart): adjust scaling margins and visible range logic

- Added `scaleMargins` property to chart config for improved chart padding.
- Updated visible range logic to extend the `from` value for better time scale adjustment.

opcode0x commented on apps/web/src/widgets/chart/ui/chartConfgs.ts:
```diff
@@ -126,5 +126,9 @@ export const timeChartOptions: DeepPartial<TimeChartOptions> = {
     autoScale: true,
     borderColor: 'transparent',
     mode: 0,
+    scaleMargins: {
```
@valeryV99 wasn't it the reason why chart breaks on some pairs?

opcode0x commented on apps/web/src/widgets/chart/logic/drawChart.ts:
```diff
@@ -90,8 +90,8 @@ export const drawChart = ({
       chart.timeScale().setVisibleLogicalRange({
-        from: visibleRange.from + 0.5,
-        to: visibleRange.to - 0.45,
+        from: visibleRange.from - 14,
```
what is 14? could you please explain a little bit here in the comment what's the purpose

valeryV99 replied:
The number 14 is the "magic" placement of ~14 bars/candles to the left. The goal is to
compensate for the autofit/scrollToRealTime, which pushes the chart to the right edge.
Or simply provide UX padding so that the current candle isn't flush with the right edge.

valeryV99 committed:
fix(chart): remove scaleMargins and introduce initialLogicalPadding

- Removed unused `scaleMargins` property from chart config.
- Introduced `initialLogicalPadding` constant for visible range logic to enhance UX.
- Updated visible range calculation to use `initialLogicalPadding`.
